extensions/amp-anim/0.1/amp-anim.js

```diff
@@ -123,6 +123,7 @@ export class AmpAnim extends AMP.BaseElement {
           }
           throw error;
         });
+    this.loadPromise_.then(() => this.updateInViewport_());
```

Comment:
For the case:  amp-anim is in viewport before `layoutCallback`, user never scrolls down. `viewportCallback` is only called once, and thus the placeholder may never be hidden even layout complete. 
I added this line to fix this kind of scenario